algorithms/PaginationHelper.js

```diff
@@ -0,0 +1,55 @@
+// For this exercise you will be strengthening your page-fu mastery.
+// You will complete the PaginationHelper class, which is a utility class helpful for querying paging information related to an array.
+// The class is designed to take in an array of values and an integer indicating how many items will be allowed per each page.
+// The types of values contained within the collection/array are not relevant.
+
+class PaginationHelper {
+    constructor(pages, pageDivider) {
+        this.pages = pages
+        this.pageDivider = pageDivider
```

Comment:
I suggest renaming to pageSize as more obvious option

Reply:
fixed